ApplySyntax.sublime-settings

```diff
@@ -73,15 +73,15 @@
             "extensions": ["html.erb"]
         },
         {
-            "syntax": "Rails/Ruby Haml",
+            "syntax": "Rails/HAML",
```

Comment:
We may need to keep the old one as a fallback for people using old versions of Sublime. At least until I'm ready to cut them off.

```js
"syntax": ["Rails/HAML", "Rails/Ruby Haml"],
```

Reply:
Ha, I didn't know this is possible, great! Thanks:) Obviously much better.
Just force-pushed with these changes.

Reply:
Perfect! I don't use Ruby, so always appreciative of those who do who are willing to help keep the rules relevant.